lib/grape/endpoint.rb

```diff
@@ -403,5 +403,9 @@ def options?
       options[:options_route_enabled] &&
         env[Grape::Http::Headers::REQUEST_METHOD] == Grape::Http::Headers::OPTIONS
     end
+
+    def method_missing(name, *args)
+      raise NoMethodError.new("undefined method `#{name}` for #{self.class}")
```

Comment:
Quote ruby style, i think it's a backtick and a quote.

```ruby
> Object.new.x
(irb):9:in `<main>': undefined method `x' for #<Object:0x0000000106b8c048>
```

Should we be replacing `self.class` with the API route?

Reply:
Sure! In fact in my first approach I had the "backtick and a quote" as you can see in the description of the PR, but finally I decided to move to the double backticks. Changing it for following ruby style :) 

About the `self.class`, totally agree on including the API route if you consider it interesting. Let me update the PR with those changes.